### DISCUSSION TITLE
Next release ETA

### DISCUSSION THREAD

**Discussion by yahavi** [docker-java/docker-java#1793] [General]
Hi, we'd like to see https://github.com/docker-java/docker-java/pull/1746 released soon.
Any ETA for the next release?
Thanks!

**Comment by eyalbe4** [comment]
Many users are affected by the late release - https://github.com/jfrog/jenkins-artifactory-plugin/issues/597 
Please assist us all by issuing a docker-java release!

**Comment by sverdlov93** [comment]
Hi, same problem here...
Is it going to be soon?
